curriculum/challenges/english/20-upcoming-python/learn-data-structures-by-building-the-merge-sort-algorithm/656709e50ed928da35260276.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 656709e50ed928da35260276
+title: Step 15
+challengeType: 20
+dashedName: step-15
+---
+
+# --description--
+
+In the previous step, you assigned the element in the `left_part` to the `array` list because it was smaller. But this will not always be the case. In some comparison cases, the element on the right could be smaller. 
```

Comment:
```suggestion
In a previous step, you assigned the element in the `left_part` to the `array` list because it was smaller. But this will not always be the case. In some comparison cases, the element on the right could be smaller. 
```